R/expand.R

```diff
@@ -28,6 +28,19 @@
 #'   `year = 2010:2020` or `year = \link{full_seq}(year,1)`.
 #'
 #'   Length-zero (empty) elements are automatically dropped.
+#' @param .name_repair Used to check that output data frame has valid
```

Comment:
Can you please use `@inheritParams` instead? (https://roxygen2.r-lib.org/articles/rd.html#inheriting-documentation-from-other-topics)

Reply:
`expand()` now inherits `.name_repair` from `expand_grid()`.